pyomo/contrib/doe/examples/fim_doe_tutorial.ipynb

```diff
@@ -737,47 +1265,118 @@
      "text": [
       "The prior information FIM:\n",
       " [[22.52943024, 1.84034314, -70.23273336, -11.09432962], [1.84034314, 18.09848116, -5.73565034, -109.15866135], [-70.23273336, -5.73565034, 218.94192843, 34.57680848], [-11.09432962, -109.15866135, 34.57680848, 658.37644634]]\n",
-      "Prior Det: 1.9558434466145787e-08\n"
+      "Prior Det: 1.9558434494323278e-08\n"
      ]
     }
    ],
    "source": [
-    "# add prior information\n",
-    "prior_pass = [[ 22.52943024 , 1.84034314, -70.23273336, -11.09432962],\n",
-    " [   1.84034314 ,  18.09848116 ,  -5.73565034 , -109.15866135],\n",
-    " [ -70.23273336 ,  -5.73565034 , 218.94192843 ,  34.57680848],\n",
-    " [ -11.09432962 , -109.15866135 ,  34.57680848 ,  658.37644634]]\n",
-    "\n",
-    "print('The prior information FIM:\\n', prior_pass)\n",
-    "print('Prior Det:', np.linalg.det(prior_pass))"
+    "# Add prior information\n",
+    "prior_pass = [\n",
+    "    [22.52943024, 1.84034314, -70.23273336, -11.09432962],\n",
+    "    [1.84034314, 18.09848116, -5.73565034, -109.15866135],\n",
+    "    [-70.23273336, -5.73565034, 218.94192843, 34.57680848],\n",
+    "    [-11.09432962, -109.15866135, 34.57680848, 658.37644634],\n",
+    "]\n",
+    "\n",
+    "# Print prior information\n",
+    "print(\"The prior information FIM:\\n\", prior_pass)\n",
+    "print(\"Prior Det:\", np.linalg.det(prior_pass))"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 20,
-   "metadata": {
-    "scrolled": false
-   },
-   "outputs": [],
+   "execution_count": 21,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "INFO: =======Iteration Number: 1 =====\n",
+      "INFO: elapsed time: 5.7\n",
+      "INFO: This is the  2 run out of  9 run.\n",
```

Comment:
I'm not sure where this info logging message is being generated but it looks like there is an off-by-one error in determining the current run number. Leads to a confusing message at the end "10 run out of 9". The grammar of this message could also be improved to "This is run 1 out of 9".

Reply:
fixed

Reply:
It doesn't look like your change to the log message was included when you reran the notebook. Are you running this with an installed version of Pyomo instead of the main branch?